po/common/en.po

```diff
@@ -279,6 +279,10 @@ msgctxt "you_can_also_help_us"
 msgid "You can also help to fund the Open Food Facts project"
 msgstr "You can also help to fund the Open Food Facts project"
 
+msgctxt "producers_administration_manual"
```

Comment:
Crowdin currently uses /po/common/common.pot as the source, so we need to copy this English entry in common.pot too.

Reply:
Done.